resources/quality/volumic/stream30ultrasc_fast.inst.cfg

```diff
@@ -0,0 +1,30 @@
+[general]
+version = 4
+name = Moyen
+definition = stream30ultrasc
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = fast
+weight = -3
+global_quality = True
+
+[values]
+layer_height = 0.15
+infill_sparse_density = 25
+top_bottom_thickness = 1.05
+wall_thickness = 1.2
+fill_outline_gaps = True
+line_width = 0.4
+retract_at_layer_change = True
```

Comment:
This setting is always true. Should be defined in the global profile instead of copypasting it.

Please check if this is also not the case for other profiles.

Reply:
all parameters have been put in global profil except two of them that change. Done in every profils.